test/regressions/index.test.ts

```diff
@@ -1,18 +1,13 @@
+import * as url from 'url';
 import * as fse from 'fs-extra';
 import { expect } from 'chai';
 import * as path from 'path';
 import * as childProcess from 'child_process';
 import { chromium } from '@playwright/test';
-import materialPackageJson from '@mui/material/package.json';
+import { version } from '@mui/material';
```

Comment:
Can avoid the string parsing if you do

```suggestion
import { major } from '@mui/material/version';
```